transformation/plugins/org.jboss.tools.fuse.transformation.editor/src/main/java/org/jboss/tools/fuse/transformation/editor/wizards/NewTransformationWizard.java

```diff
@@ -85,6 +83,7 @@
 
     public static final String CAMEL_CONFIG_PATH = MavenUtils.RESOURCES_PATH + "META-INF/spring/camel-context.xml"; //$NON-NLS-1$
     private static final String OBJECT_FACTORY_NAME = "ObjectFactory"; //$NON-NLS-1$
+    private static final String CAMEL_PKG = "org.apache.camel"; //$NON-NLS-1$
```

Comment:
Optional: what do you think to call it CAMEL_GROUPID ? if I understand well the code, this will be more accurate.